### PR TITLE
Math-Magicians, PR for creating the Calculator UI

### PR DESCRIPTION
**Pull request by Miliyonayalew**
## In this milestone i
- Used React Components
- Created a directory called `components` and create `Calculator.js` file
- Developed the required design in `Calculator.js` component.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (DeliceLydia) [inline src/App.js]: - [ ] Please make this function a class

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Miliyonayalew, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.js (hunk 1 above)
Comment (Miliyonayalew) [reply src/App.js]: Hey, @DeliceLydia Thank you for the review, 
- as the requirement says I use class-based components and Once the Calculator.js component is ready, import it and use it in your main component (App.js).

**Review by t-yanick — APPROVED** [review]
Hi @Miliyonayalew , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Miliyonayalew** [comment]
@t-yanick  Thank you for the review, Happy coding 😄

**Pull request merged**